src/frankenphp-symfony/src/Runner.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Runtime\FrankenPhpSymfony;
+
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpKernel\HttpKernelInterface;
+use Symfony\Component\HttpKernel\TerminableInterface;
+use Symfony\Component\Runtime\RunnerInterface;
+
+/**
+ * A runner for FrankenPHP.
+ *
+ * @author Kévin Dunglas <kevin@dunglas.dev>
+ */
+class Runner implements RunnerInterface
+{
+    public function __construct(private readonly HttpKernelInterface $kernel)
+    {
+    }
+
+    public function run(): int
+    {
+        $server = array_filter($_SERVER, static fn (string $key) => !str_starts_with($key, 'HTTP_'), ARRAY_FILTER_USE_KEY);
+        do {
+            $ret = frankenphp_handle_request(function () use ($server, &$sfRequest, &$sfResponse) {
+                // Merge the environment variables coming from DotEnv with the one tight to the current request
+                $_SERVER += $server;
+
+                $sfRequest = Request::createFromGlobals();
+                $sfResponse = $this->kernel->handle($sfRequest);
+
+                $sfResponse->send();
+            });
+
+            if ($this->kernel instanceof TerminableInterface && $sfRequest && $sfResponse) {
+                $this->kernel->terminate($sfRequest, $sfResponse);
+            }
+        } while ($ret);
```

Comment:
Sorry for the review but maybe the callable could define its return type `: void` if I'm right, same for the first one: 

```php
$server = array_filter($_SERVER, static fn (string $key): bool => !str_starts_with($key, 'HTTP_'), ARRAY_FILTER_USE_KEY);
```

I'm not sure if the current lib tend to define return types on callable 🙁

Reply:
I usually dont add return types on callables. It gives zero to little benefit. But it is just a matter of taste. As long as CI is not complaining, Im fine with adding return types.